db/migrate/20200312144826_update_existing_unavailable_fors.rb

```diff
@@ -0,0 +1,14 @@
+class UpdateExistingUnavailableFors < ActiveRecord::Migration[6.0]
+  def up
+    expiration_length = Config.get(:unavailable_for_length)
+    return if expiration_length.zero?
+
+    UnavailableAsCandidateFor.find_each do |u|
+      u.update(expires_at: u.created_at + expiration_length.days)
+    end
+
+    Client.where("prevent_matching_until > ?", Date.current).find_each do |client|
```

Comment:
Seems to me that we should either move all parked client, not just the future ones... If there are expired parked clients, should we be cleaning them up?

Reply:
Those should have been cleaned up already.  There's a nightly process that takes care of that currently.  Also, we shouldn't be looking at the field anymore, and it should probably be removed eventually.

Reply:
If they are cleaned up, can we just do `Client.where.not(prevent_matching_until: nil)`?